pkgs/by-name/cd/cddl/Gemfile

```diff
@@ -1,2 +1,4 @@
 source 'https://rubygems.org'
 gem 'cddl'
+
+gem "base64", "~> 0.2.0"
```

Comment:
Since this PR was opened, 0.3.0 has been released.

Maybe also mention that this is for compatibility with Ruby 3.4?

(Ideally, this would be upstreamed, but the repo mentioned in the gem https://github.com/cabo/cddl isn't available.)